### DISCUSSION TITLE
Does sleap work on different aninmals?

### DISCUSSION THREAD

**Discussion by miraclezero** [talmolab/sleap#1232] [Ideas]
Only can have one skeleton now. Is it possible to add another skeleton for a different animal in the same frame?

**Comment by talmo** [comment]
Hi @miraclezero,

Not at the moment. You'll want to create separate projects each with their own skeleton and basically just run each of the species-specific models on the same video.

Let us know if you have any questions!

Talmo

**Reply by miraclezero** [reply]
Ok, thanks
